### PR TITLE
More ruins

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request
Adds even more ruins
Moves oldstation to icemoon, with some goodies
Adds a new space ruin: the solgov history musuem
Makes a new ash drake ruin
Ports the Chaos marne megafauna by EgorDinamit, but de warhammered the fauna to prevent any 'intresting' types from taking intrest
https://github.com/vlggms/fulpstation/pull/3
https://github.com/vlggms/tegustation/pull/136
Ports https://github.com/tgstation/tgstation/pull/50813/files
and edits it to be only on one z level
Ports
https://github.com/Kitsunemitsu/KirieStation/pull/111
https://github.com/Kitsunemitsu/KirieStation/pull/123
https://github.com/Kitsunemitsu/KirieStation/pull/220

gate sound effect is from [here](https://freesound.org/people/craigsmith/sounds/483228/)
## Why It's Good For The Game

i had these around

## Changelog
:cl:
add: Adds a new ruin: a history musuem!
add: Moves oldstation t- you know, i dont even think these changelogs work
/:cl:

### CODE REVIEW

**Comment by triplezeta** [comment]
terragov pulse rifle :/

**Comment by rye-rice** [comment]
skill issue

**Comment by HighPrincessErinys** [comment]
zeta it uses terragov shit because terragov is old and its a history museum dont you know what a museum is you really are a stupid moth whore zeta i really cannot believe you are this stupid you are a bumbling idiot

**Comment by ZephyrTFA** [comment]
Let's not insult people on the GH, even as a joke.

**Comment by HighPrincessErinys** [comment]
literally 1827

**Comment by rye-rice** [comment]
literally 2487

**Comment by Valorium-RB** [comment]
Can we get some pictures of the ruins in question?

**Comment by rye-rice** [comment]
Maybe

**Comment by rye-rice** [comment]
Linters are mad, rockplanet will be in another PR, otherwise this should be ready

**Comment by ZephyrTFA** [comment]
This isn't ready to merge. 
You need to read the linters not just ignore them

**Comment by ZephyrTFA** [comment]
```
code/game/objects/structures/ghost_role_spawners.dm, line 826, column 57:
error: proc never calls parent, required by /atom/proc/Initialize
- code/game/atoms.dm:204:2: required by this must_call_parent annotation

code/game/objects/structures/ghost_role_spawners.dm, line 889, column 58:
error: proc never calls parent, required by /atom/proc/Initialize
- code/game/atoms.dm:204:2: required by this must_call_parent annotation

code/game/objects/structures/ghost_role_spawners.dm, line 911, column 53:
error: proc never calls parent, required by /atom/proc/Initialize
- code/game/atoms.dm:204:2: required by this must_call_parent annotation

code/game/objects/structures/ghost_role_spawners.dm, line 844, column 51:
error: proc never calls parent, required by /atom/proc/Initialize
- code/game/atoms.dm:204:2: required by this must_call_parent annotation

code/game/objects/structures/ghost_role_spawners.dm, line 863, column 54:
error: proc never calls parent, required by /atom/proc/Initialize
- code/game/atoms.dm:204:2: required by this must_call_parent annotation

code/game/objects/structures/ghost_role_spawners.dm, line 951, column 50:
error: proc never calls parent, required by /atom/proc/Initialize
- code/game/atoms.dm:204:2: required by this must_call_parent annotation

code/game/objects/structures/ghost_role_spawners.dm, line 933, column 53:
error: proc never calls parent, required by /atom/proc/Initialize
- code/game/atoms.dm:204:2: required by this must_call_parent annotation
```

**Comment by rye-rice** [comment]
fixing

**Comment by rye-rice** [comment]
fixed

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 1]
```diff
@@ -813,7 +813,18 @@
...
 
 //ashdrake lair ghost roles
-/obj/effect/mob_spawn/human/doctor/alive/lost
+/obj/effect/mob_spawn/human/lost
+	death = FALSE
+	roundstart = FALSE
+	random = TRUE
+
+/obj/effect/mob_spawn/human/lost/Initialize(mapload)
+	..()
```
Comment (ZephyrTFA) [inline code/game/objects/structures/ghost_role_spawners.dm]:
```suggestion
. = ..()
```

**Pull request merged**